### ISSUE TITLE
Transform scale on Parent dom element get's positions out of sync

### ISSUE THREAD

**Issue by vikasaditya** [framer/motion#265]
Transform scale on Parent dom element get's positions out of sync

1. Go to  https://codesandbox.io/s/drag-7qcc8
Two motion.div examples have scale at 0.5 and 1.5. 
drag the inner, we are not taking the transformed into consideration.

Is this something we can fix while using Drag & constraints?

**Comment by InventingWithMonster** [comment]
Motion already supports an architecture that does something similar in Framer X. I can investigate what it'd take to do something similar here.

**Comment by vikasaditya** [comment]
hi, any pointer on what to try?

**Comment by InventingWithMonster** [comment]
Hey @vikasaditya in case this ticket is still relevant to you, here's an example setup https://codesandbox.io/s/drag-forked-byi3k?file=/src/index.tsx

`transformPagePoint` takes the point measured relative to the viewport and needs to return it untransformed. The exact method is up to you but I've included some notes in the Sandbox.

**Comment by spadgeaki** [comment]
Hello, I have a same case as @vikasaditya. I tried what @mattgperry recommended but I am confused - function provided to `transformPagePoint` prop doesn't seem to get called. And I didn't find anything regarding its use in documentation.

I edited official example of `AnimateSharedLayout` to demonstrate the issue with scaled parent:
https://codesandbox.io/s/framer-motion-2-animatesharedlayout-animate-between-different-components-forked-bbwyq?file=/src/App.js

Thank you for any help.
